### PR TITLE
Feat cards

### PR DESCRIPTION
**Pull request by hinanshisuthar**
Card component created
- Cards with badges
- Cards with dismiss
- Cards with text overlay
- Text only cards
- Vertical/Horizontal cards
- Cards with shadow

https://user-images.githubusercontent.com/88201664/152704592-602d0736-e60e-4fcc-99d4-243d9a0ae2d4.mp4

### CODE REVIEW

File: components/cards.css [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+    justify-content: space-between;
+    align-items: baseline;
+    width: 100%;
+}
+
+.btn-badge {
+    color: var(--background-color);
+    font-size: smaller;
+    border-radius: 5px;
+    font-weight: 600;
```
Comment (dev-enforced) [inline components/cards.css:61]: Suggestion:
You can store this font-weight as a variable in root as font-semi-bold if this value is used frequently throughout multiple files.

File: components/cards.css [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+.card-horizontal .product {
+    flex-basis: 20%;
+}
+
+.card-horizontal .prod-links {
+    flex-grow: 1;
+}
+
+.card-horizontal .prod-links .prod-content .prod-info {
+    justify-content: flex-start;
```
Comment (dev-enforced) [inline components/cards.css:206]: You can use this as a utility class named as .native-justify-flex-start with these properties and pass it into the HTML wherever needed!!

**Review by dev-enforced — APPROVED** [review]
Documentation of cards is really great @hinanshisuthar and of course the way they all look giving a really good feel of the shopping website cards. Passed some suggestions which if needed can be added onto your code and then merged!!

File: components/cards.css [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+    border-radius: 8px;
+}
+
+.prod-container {
+    position: relative;
+}
+
+.prod-container img {
+    max-width: 100%;
+    height: auto;
```
Comment (MaruthiPothuganti) [inline components/cards.css:30]: Typo: Maybe here you wanted to use max-height property...

**Review by MaruthiPothuganti — COMMENTED** [review]
Recommended changes: typo in a height property

File: components/cards.css (hunk 3 above)
Comment (hinanshisuthar) [reply components/cards.css:30]: No. This is for the responsive image. I think max-width and height auto go together right?

Comment (MaruthiPothuganti) [reply components/cards.css:30]: I was saying that you are using height property two times with different values _height: auto;_  and   _height: 100%;_

**Pull request merged**